src/library/simplified/extension/number-of-guests.js

```diff
@@ -127,9 +124,9 @@
             b['numberOfGuests'] = b['entity.checkout.activity.travelerInfo.numberOfTickets'];
         }
     }
-	else if (utag.isCruiseCabinN() && b['entity.cruise.passengerCount'])
+	else if (utag.isCruiseCabinN() && b.entity.cruise.traveler)
```

Comment:
basically, if b['entity.cruise.traveler.totalNumberOfTravelers'] exists then retrieve the value.